universite-du-quebec-a-montreal.csl

```diff
@@ -5,7 +5,7 @@
     <title-short>UQAM</title-short>
```

Comment:
@jjrondeau, I assume you'd like to change the title to `<title>Université du Québec à Montréal - APA (Français - Canada)</title>` as well?

To continue our discussion from https://github.com/citation-style-language/styles/issues/5401, in general it's better to just leave the style ID as-is, as most users will just look at the style title and probably never notice the style ID, and changing the ID has some significant downsides.

Reply:
Yes, exactly. Changing only the title would be fine. Style ID (and self link?) can stay unchanged if this is possible, although the Visual CSL Editor warn me it doesn't respect style repository convention.
Thank you.